projects/addon-mobile/components/sheet/directives/sheet-wrapper/sheet-wrapper.directive.ts

```diff
@@ -37,18 +36,18 @@ export class TuiSheetWrapperDirective {
     @ContentChild(TuiSheetComponent, {read: TUI_SHEET_SCROLL})
     private readonly scroll$!: Observable<number>;
 
+    @Input()
+    tuiSheetWrapper = 16;
```

Comment:
nit: not very clear name (wrapper = number), maybe tuiSheetWrapperOffset or tuiSheetOffset?